pandas/tests/frame/common.py

```diff
@@ -1,3 +1,6 @@
+from contextlib import contextmanager
```

Comment:
Need to revert changes to this file (will fail CI)

Reply:
Removed in 3190b0f1fd6863b565be1e634a77e8ed429c2fa4